frontend/packages/network-attachment-definition-plugin/src/models/index.ts

```diff
@@ -34,4 +36,5 @@ export const HyperConvergedModel: K8sKind = {
   abbr: 'SRNNPM', // TODO check on this
```

Comment:
This looks wrong :)

Reply:
haha... it is. I've already got a PR to fix it. https://github.com/openshift/console/pull/3202